docs/pages/versions/unversioned/guides/push-notifications.md

```diff
@@ -136,14 +136,14 @@ export default class AppContainer extends React.Component {
 
 ### Notification handling timing
 
-It's not entirely clear from the above when your app will be able to handle the notification depending on it's state at the time the notification is received. For clarification, see the following table:
+It's not entirely clear from the above when your app will be able to handle the notification depending on it's state at the time the notification is received. For clarification, see the table below. `Exponent.notification` refers to a notification that can be picked up using `Notifications.addListener`, as shown in the example above.
```

Comment:
I don't think there's any reason for this to be called `Exponent.notification` anymore -- this is a really old relic 😮 Could we just change this wording to something like "Listener called with notification: origin: ..." instead?